src/vm/i386/asmhelpers.S

```diff
@@ -815,6 +815,13 @@ PATCH_LABEL ExternalMethodFixupPatchLabel
 NESTED_END ExternalMethodFixupStub, _TEXT
 
 #ifdef FEATURE_READYTORUN
+NESTED_ENTRY DynamicHelperArgsStub, _TEXT, NoHandler
+    CHECK_STACK_ALIGNMENT
+    call  eax
+    add   esp, 12
+    ret
+NESTED_END DynamicHelperArgsStub, _TEXT
```

Comment:
As I understand, unwind may failed to unwind this function. The following change will make this function unwindable:
```
mov  [esp + 8], ebp
lea ebp, [esp +8]
CHECK_STACK_ALIGNMENT
call eax
mov esp, ebp
pop ebp
ret
```
@janvorli Could you please take a look?